app/javascript/components/FiltersBar/FiltersBar.jsx

```diff
@@ -24,44 +27,86 @@ SelectInput.propTypes = {
   options: PropTypes.arrayOf(PropTypes.string),
 };
 
-export default function FiltersBar({ cuisins, filterCuisins, filterRating, filterSpeed }) {
-  return (
-    <div className="row py-3 bg-danger">
-      <div className="container">
-        <div className="row py-2">
-          <div className="col-12 col-sm-4">
-            <SelectInput
-              label="Cuisins"
-              options={cuisins.map(cuisin => cuisin)}
-              handleChange={filterCuisins}
-              defaultOption="Sushi, israeli, oriental..."
-            />
-          </div>
-          <div className="col-12 col-sm-4">
-            <SelectInput
-              label="Rating"
-              options={ratings.map(rate => `${rate} ${rate === 1 ? 'star' : 'stars'}`)}
-              handleChange={filterRating}
-              defaultOption="I'm an all-star"
-            />
-          </div>
-          <div className="col-12 col-sm-4">
-            <SelectInput
-              label="Speed"
-              options={deliveryTimes.map(minutes => `Up to ${minutes} minutes`)}
-              handleChange={filterSpeed}
-              defaultOption="I need it before..."
-            />
+class FiltersBar extends Component {
+  componentDidMount() {
```

Comment:
consider `componentWillMount` as it will be dispatched sooner increasing perceived responsiveness